### ISSUE TITLE
 Default showCursor to true in ShadInputFormField

### ISSUE THREAD

**Issue by veluloganathan** [nank1ro/flutter-shadcn-ui#210]
### Steps to reproduce

Currently, the ShadInputFormField widget has its showCursor property defaulted to false. This behavior can cause confusion for users expecting the text cursor to be visible by default, especially in editable text fields. Having showCursor default to true would align with the typical behavior of text input fields in most UI frameworks.

### Expected results

ShadInputFormField(
  controller: controller,
  // No need to specify `showCursor: true` explicitly
);

### Actual results

ShadInputFormField(
  controller: controller,
   showCursor: true,
);

### shadcn_ui version

0.16.0

### Platform

Web

### Code sample

```
ShadInputFormField(
  controller: controller,
   showCursor: true,
);
```

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by nank1ro** [comment]
fixed in the v0.16.3

**Comment by veluloganathan** [comment]
Super fast! Glad 👍
